### PR TITLE
Use arm rest api instead of kudu for webjobs

### PR DESCRIPTION
**Pull request by nturinski**
Related to https://github.com/Microsoft/vscode-azureappservice/issues/891 but does not fix the issue

### CODE REVIEW

File: appservice/src/SiteClient.ts [hunk 1]
```diff
@@ -259,4 +259,10 @@ export class SiteClient {
     public async getPublishingUser(): Promise<User> {
         return await this._client.getPublishingUser({});
     }
+
+    public async getWebJobs(): Promise<WebJobCollection> {
```
Comment (EricJizbaMSFT) [inline appservice/src/SiteClient.ts]: nit: Prefer to keep the naming consistent with the below call (listWebJobs)

File: appservice/src/SiteClient.ts [hunk 2]
```diff
@@ -259,4 +259,10 @@ export class SiteClient {
...
+        return this.slotName ?
+            await this._client.webApps.listWebJobsSlot(this.resourceGroup, this.siteName, this.slotName) :
+            await this._client.webApps.listWebJobs(this.resourceGroup, this.siteName);
```
Comment (EricJizbaMSFT) [inline appservice/src/SiteClient.ts]: Do you need to add a listWebJobsNext? Does that exist?

**Comment by nturinski** [comment]
It seems to work the same as `kudu.listJobs` did, but the source of the problem is that WebJobs isn't supported on Linux and returns 404 instead.

https://github.com/Microsoft/vscode-azureappservice/pull/934

**Comment by EricJizbaMSFT** [comment]
Okay cool as long as it works on windows

File: appservice/src/SiteClient.ts (hunk 2 above)
Comment (nturinski) [reply appservice/src/SiteClient.ts]: Does not exist

Comment (nturinski) [reply appservice/src/SiteClient.ts]: Ugh, sorry this is incorrect.  It wasn't showing up before, but I see it in the API.

**Pull request merged**